Components/Avatar/Avatar.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../../css/header.css" />
+    <link rel="stylesheet" href="../../css/sidebar.css" />
+    <link rel="stylesheet" href="../../css/content.css" />
+    <link rel="stylesheet" href="../../css/divider.css" />
+    <link rel="stylesheet" href="../../css/container.css" />
+    <link rel="stylesheet" href="../../css/media-query.css">
```

Comment:
You are linking every html file with so many link tags I want to suggest you something like import these all css file in a single css file and then link that css file to every html that way you only link one css to evert html file and it will decrease complexity and increase reusability. This is my suggestion

Reply:
thanks for the suggestion akshat. I will do this changes once all my code in merged in dev branch as it will be easy to modify